### PR TITLE
[Bug] Slash command in query pane

### LINKED ISSUE #5664: [Bug] Slash commands do not work in query/api pane
## Description
`/` commands should work in the query/api pane. Although, the auto suggestion box should only popup if user enters `/` or clicks on `/` btn

### Steps to reproduce the behaviour:
1. Create a query
2. try entering `/`

### Important Details
- Version: [Cloud / Self-Hosted]
- OS: [MacOSX]
- Browser [chrome, safari]
- Environment [release]

### PR DESCRIPTION
**Pull request by pranavkanade**
## Description
Reactivated the slash command in query/api pane. The suggestion box would only show up if user manually enters `/` or clicks on `/` button.

Fixes #5664

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
manually

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>

    // Code coverage diff between base branch:release and head branch: bug/slash-command-in-query-pane 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | app/client/src/components/editorComponents/CodeEditor/commandsHelper.ts | 42.11 **(1.04)** | 40 **(-1.67)** | 22.22 **(0)** | 44.23 **(1.09)**
 :green_circle: | app/client/src/components/editorComponents/CodeEditor/generateQuickCommands.tsx | 17.24 **(0.29)** | 0 **(0)** | 0 **(0)** | 15.79 **(0.27)**
 :red_circle: | app/client/src/components/editorComponents/CodeEditor/index.tsx | 72.96 **(0)** | 54.96 **(-0.08)** | 52.78 **(0)** | 73.64 **(0)**</details>

### CODE REVIEW

File: app/client/src/components/editorComponents/CodeEditor/index.tsx [hunk 1]
```diff
@@ -416,7 +418,7 @@ class CodeEditor extends Component<Props, State> {
     });
     this.setState({ isFocused: true }, () => {
       if (preventAutoComplete) return;
-      this.handleAutocompleteVisibility(this.editor);
+      this.handleAutocompleteVisibility(this.editor, true);
```
Comment (ApekshaBhosale) [inline app/client/src/components/editorComponents/CodeEditor/index.tsx:421]: is this supposed to be always true?

**Review by ApekshaBhosale — CHANGES_REQUESTED** [review]

File: app/client/src/components/editorComponents/CodeEditor/index.tsx (hunk 1 above)
Comment (pranavkanade) [reply app/client/src/components/editorComponents/CodeEditor/index.tsx:421]: If user clicks on `/` btn, then the auto suggestion box has to appear, so yes.

**Review by ApekshaBhosale — DISMISSED** [review]

**Review by ApekshaBhosale — APPROVED** [review]

**Pull request merged**